client/src/lib.js

```diff
@@ -44,7 +44,7 @@ class NFTStorage {
    *
    * @param {{token: string, endpoint?:URL}} options
    */
-  constructor({ token, endpoint = new URL('https://nft.storage') }) {
+  constructor({ token, endpoint = new URL('https://api.nft.storage') }) {
```

Comment:
⚠️ We need to redirect the old URLs in cloudflare for old clients.

Reply:
already done for production